packages/adapter/src/methods.ts

```diff
@@ -162,4 +162,17 @@ export async function getAllTransactions(pluginOrigin: string, address?: string)
       }
     }]
   });
+}
+
+export async function sendUnit(pluginOrigin: string, amount: string | number, to: string): Promise<unknown> {
+  return await window.ethereum.send({
+    method: pluginOrigin,
+    params: [{
+      method: 'sendUnit',
+      params: {
+        amount, 
+        to
+      }
+    }]
+  });
```

Comment:
- return type is string
- use `sendSnapMethod` to insure typesafety and code reusage
- you can refactor rest of method to use `sendSnapMethod` instead of using window.ethereum directly